TRADEMARKS.md

```diff
@@ -0,0 +1,231 @@
+**AmpliPi™** is a trademark of **MicroNova LLC**. You may not use this trademark without prior express written permission of MicroNova LLC and pursuant to the trademark policy shown below. This unmodified file must be included with all copies of the AmpliPi software.
+
+Version 1.0  
+Last Modified: March 8, 2021
+
+# Table of Contents
+
+- [Introduction](#Introduction)
+- [Our Commitment to Open Source Principles](#Our-Commitment-to-Open-Source-Principles)
+- [Trademarks Subject to the Guidelines](#Trademarks-Subject-to-the-Guidelines)
+- [Universal Considerations for All Uses](#Universal-Considerations-for-All-Uses)
+- [Use for Software](#Use-for-Software)
+- [Use for Non-Software Goods and Services](#Use-for-Non-Software-Goods-an-Services)
```

Comment:
#Use-for-Non-Software-Goods-an-Services --> #Use-for-Non-Software-Goods-and-Services